__tests__/app.test.js

```diff
@@ -127,8 +128,60 @@ describe('api', () => {
     return request(app)
       .get('/api/reviews/bananas/comments')
       .expect(400)
-      .then(({body:{msg}}) => {
+      .then(({ body: { msg } }) => {
+        expect(msg).toBe('Invalid id')
+      })
+  })
+  test('201: POST request responds with comment object', () => {
```

Comment:
Worth one more 201 test. Does it ignore unnecessary properties on the request comment?